src/fiuba/algo3/tp2/PicoFino.java

```diff
@@ -1,11 +1,22 @@
 package fiuba.algo3.tp2;
 
-import fiuba.algo3.tp2.estrategiasDesgaste.EstrategiaDesgasteDividida;
+import fiuba.algo3.tp2.estrategiasDesgaste.EstrategiaDesgasteDivididaDeDurabilidad;
 
 public class PicoFino extends Pico {
-    public PicoFino() {
+
+    public PicoFino(){
+
         this.durabilidad = 1000;
         this.fuerza = 20;
-        this.desgastador = new EstrategiaDesgasteDividida(this.durabilidad, this.fuerza, 0.1);
+        this.desgastador = new EstrategiaDesgasteDivididaDeDurabilidad(this.durabilidad,10);
+    }
+
+    @Override
+    public void romper(Material unMaterial) {
```

Comment:
Che podemos hacer esto para no usar el instanceof.

Hace un romper especifico para diamante que haga this.desgastador.desgastar()

Y un @override de romper(Material unMaterial) que solo haga un material.desgastar(this) sin llamar al desgastador de la herramienta

Reply:
tendrias algo asi:
`
    public void romper(DiamanteMaterial unDiamante) {
        unMaterial.desgastar(this);
        this.desgastador.desgastar();
   }
@override
public void romper(Material unMaterial) {
        unMaterial.desgastar(this);
    }
`

Reply:
Me parece bien eso!